### PR TITLE
Security documentation

### PR DESCRIPTION
**Pull request by sidohaakma**
I restructured the current user documentation to make it clear that the entry point is now the creation of groups.

Also fixed a number of references to other files. 

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- (n.a.) Code unit/integration/system tested
- [x] User documentation updated
- (n.a.) (If you have changed REST API interface) view-swagger.ftl updated
- (n.a.) Test plan template updated
- (n.a.) Clean commits
- (n.a.) Added Feature/Fix to release notes

### CODE REVIEW

File: docs/user_documentation/admin-features/guide-ref-user-management.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Note that it is not possible to delete users.
+You can only set them to inactive, which will prevent them from logging in.
+
+## Authentication
+
+To configure how to authenticate in MOLGENIS you can read [configure authentication](guide-ref-authentication.md).
+
+## Permissions
+ 
+You can lso define more fine grained permissions to each user by setting the permission on users instead of
```
Comment (bartcharbon) [inline docs/user_documentation/admin-features/guide-ref-user-management.md]: "lso" typo

File: docs/user_documentation/admin-features/guide-ref-security.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+* **Plugin**
+  
+  Plugins are specific modules within MOLGENIS that offer certain functionality. For example the [Data Explorer](../finding-data/guide-explore.md).
+      
+## Add groups
+*note: you need to be superuser to add groups*
+
+You can add groups in MOLGENIS by navigating to **Admin --> Security Manager**. 
+
+![Security Manager menu](../../images/security/main_menu_security.png?raw=true, "Security Manager menu")
```
Comment (tommydeboer) [inline docs/user_documentation/admin-features/guide-ref-security.md]: Remove the Chrome bar from this screenshot, it contains your name :wink:.
Screenshot bonus points: hover over 'Security Manager' in the dropdown menu for maximum clarity.

File: docs/user_documentation/admin-features/guide-ref-security.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
+# Setup security in MOLGENIS
```
Comment (tommydeboer) [inline docs/user_documentation/admin-features/guide-ref-security.md]: Screenshots should only show the important bits, I think. Right now the images in this section contain too much white space for my taste. Also, the menu bar can be removed in most of the images.

File: docs/user_documentation/admin-features/guide-configure-molgenis.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+**
```
[the remark this answers is not in the document]
Comment (sidohaakma) [reply docs/user_documentation/admin-features/guide-configure-molgenis.md]: Move ref docs to top of the short summary page

**Pull request merged**